nl-be/auth.json

```diff
@@ -0,0 +1,284 @@
+            "invitation_description": "U bent uitgenodigd om lid te worden van ${business_name}. Login of maak een account aan om de uitnodiging te accepteren.",
+            "disclaimer_clickwrap_input_label": "Voorwaarden accepteren",
+            "disclaimer_clickwrap_error_required": "U moet de voorwaarden lezen en accepteren om verder te kunnen gaan",
+            "disclaimer_dialog_heading": "Accepteren om door te gaan",
+            "disclaimer_dialog_description": "Om door te gaan met aanmelden, moet u onze voorwaarden lezen en accepteren",
+            "disclaimer": "Door verder te gaan, gaat u akkoord met onze voorwaarden",
+            "disclaimer_terms_of_use_link": "Gebruiksvoorwaarden",
+            "disclaimer_privacy_link": "Privacyverklaring"
+        },
+        "sms_verify_code_page": {
+            "page_title": "Multi-factor verificatie | ${business_name}",
+            "heading": "Telefoonnummer verifiëren",
+            "description": "Voer de code in die we zojuist hebben verzonden naar ${telefoon_nummer].",
+            "code_input_label": "Code",
+            "code_input_error_expired_code": "Uw bevestigingscode is verlopen, vraag een nieuwe aan",
+            "code_input_error_incorrect_code": "Voer een geldige bevestigingscode in",
+            "code_input_error_incorrect_code_length": "De bevestigingscode moet uit 6 cijfers bestaan",
+            "code_input_error_resend_requested_too_soon": "Kan nog geen code versturen. Probeer het later nog eens",
+            "confirmation_code_input_error_unknown_error": "Fout met verificatie",
+            "continue_button": "Controleer",
+            "didnt_receive_a_code_description": "Geen code ontvangen?",
+            "didnt_receive_a_code_link": "Code opnieuw verzenden",
+            "choose_another_method_link": "Kies een andere methode"
+        },
+        "subscribe_form_widget": {
+            "page_title": "Aanmelden",
+            "heading": "Blijf op de hoogte",
+            "description": "Blijf op de hoogte van productupdates, nieuws en evenementen",
+            "name_input_label": "Voornaam",
+            "name_input_error_required": "Voer een naam in,",
+            "name_input_error_exceeds_max_length": "Naam is te lang,",
+            "email_input_label": "E-mail",
+            "email_input_error_invalid": "Voer een geldig e-mailadres in,",
+            "email_input_error_required": "E-mail is verplicht,",
+            "email_input_error_duplicate_email_found": "Dit e-mailadres is al gebruikt,",
+            "continue_button": "Registreren",
+            "success_heading": "Succes! Welkom bij de loop.",
+            "credit_powered_by": "Aangedreven door"
+        },
+        "verification_code_sms": {
+            "description": "${generated_code} is uw eenmalige code om u aan te melden bij ${business_name} @${receiver_domain} #${generated_code}"
```

Comment:
@onderay can we add two `\n\n` to start a new line in the SMS between `${business_name} @${receiver_domain}` so `${business_name}\n\n@${receiver_domain}`

Reply:
@DaveOrDead updated